File: results/java/CALCULATE_SUM_OF_ALL_NUMBERS_PRESENT_IN_A_STRING.cpp
int f_filled ( string str ) {
  string temp ;
  int sum = 0 ;
  for ( int i = 0 ;
  i < str . length ( ) ;
  i ++ ) {
    char ch = str [ i ] ;
    if ( isdigit ( ch ) ) {
      temp += ch ;
    }
    else {
      sum += atoi ( temp . c_str ( ) ) ;
      temp = "0" ;
    }
  }
  return sum + atoi ( temp . c_str ( ) ) ;
}

File: results/java_codex/CALCULATE_SUM_OF_ALL_NUMBERS_PRESENT_IN_A_STRING.java
int findSum ( string str ) {
  string temp = "";
  int sum = 0;
  for ( int i = 0;
  i < str . length ( );
  i ++ ) {
    char ch = str [ i ];
    if ( isdigit ( ch ) ) temp += ch;
    else {
      sum += stoi ( temp );
      temp = "0";
    }
  }
  return sum + stoi ( temp );
}